index.js

```diff
@@ -88,7 +88,7 @@ const waitForElementNotToBeVisible = function(
   browser.wait(EC.invisibilityOf(htmlElement), timeout, message);
 };
 
-const waitForElementToBeClickable = function(
+function waitForElementToBeClickable(
```

Comment:
As I said in the previous comment, put this right after the `getDefaultIsNotClickableMessage` function. In other words, put this method on line 33 of the `index.js` file.

Reply:
Also, there is no need for the semi-colon at the end of this function.

Reply:
So much adjusts hahaha, sorry for that.